astropy/utils/tests/test_data.py

```diff
@@ -1140,17 +1140,14 @@ def osraiser(dirnm, linkto, pkgname=None):
 def test_read_unicode(filename):
     contents = get_pkg_data_contents(os.path.join("data", filename), encoding="utf-8")
     assert isinstance(contents, str)
-    contents = contents.splitlines()[1]
-    assert contents == "האסטרונומי פייתון"
+    assert contents.splitlines()[0] == "האסטרונומי פייתון"
```

Comment:
This works even on Windows:
```suggestion
    assert contents == "האסטרונומי פייתון\n"
```